assets/js/components/post-searcher.js

```diff
@@ -168,11 +185,13 @@ class PostSearcher extends Component {
 										<Button
 											onClick={ this.onClick }
 											className="googlesitekit-post-searcher__button"
+											disabled={ ! this.state.canSubmit }
 										>
 											{ __( 'View Data', 'google-site-kit' ) }
 										</Button>
 									</div>
 								</div>
+								{ 0 < this.state.message.length && <p>{ this.state.message }</p> }
```

Comment:
This shouldn't be added as the only visual change which has been approved is the disabled state of the submit button. Originally the issue mentioned adding feedback, but this was removed before execution 🙂

Reply:
Updated.